modules/update-best-practices.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-best-practices_{context}"]
+= Best practices for cluster updates
+
+{product-title} is designed to provide a robust update experience that allows clusters to update with minimal disruptions to workloads.
+Updates will not begin unless the cluster is determined to be in an upgradeable state at the time of the update request.
+
+This design ensures that some key conditions are met before initiating an update, but there are a number of actions you can take to increase your chances of a successful cluster update.
+
+[id="recommended-versions_{context}"]
+== Choose versions recommended by the OpenShift Update Service
+
+The OpenShift Update Service (OSUS) provides update recommendations based on cluster characteristics such as the cluster's subscribed channel, which are then saved by the Cluster Version Operator as either recommended or conditional updates.
+While it is possible to attempt an update to a version that is not recommended by OSUS, following a recommended update path protects users from encountering known issues or unintended consequences on the cluster.
+
+Choose only update targets that are recommended by OSUS to ensure a successful update.
+
+[id="critical-alerts_{context}"]
+== Address all critical alerts on the cluster
+
+Critical alerts must always be addressed as soon as possible, but it is especially important to address these alerts and resolve any problems before initiating a cluster update.
+Failing to address critical alerts before beginning an update can cause problematic conditions for the cluster.
+
+In the *Administrator* perspective of the web console, critical alerts can be
+found in *Observe -> Alerting*.
+
+[id="cluster-upgradeable_{context}"]
+== Ensure that the cluster is in an Upgradable state
+
+When one or more Operators have not reported their `Upgradeable` condition as `True` for more than an hour, the `ClusterNotUpgradeable` warning alert is triggered in the cluster.
+In most cases patch updates are not blocked by this alert, but you cannot perform a minor version update until this alert is resolved and all Operators report `Upgradeable` as `True`.
+
+For more information about the `Upgradeable` condition, see "Understanding cluster Operator condition types" in the additional resources section.
+
+[id="nodes-ready_{context}"]
+== Ensure that enough spare nodes are available
+
+// Completely guessing the explanation in this section just to have something to start with when this is reviewed by an SME.
+A cluster should not be running with little to no spare node capacity, especially when initiating a cluster update.
+Nodes that are not running and available may limit a cluster's ability to perform an update with minimal disruption to cluster workloads.
```

Comment:
I think it would be better to reword this in a positive voice, such as "You should have spare node capacity to ensure.." and "Having available nodes is important for cluster updates to complete with minimal disruption" or something to that effect. I'm not entirely sure what is meant by "running and available" though.

Reply:
I'll admit this sounds negative but I think it's useful to talk about the consequences this can have on update success, especially since there's already a more positive sounding, actionable piece of advice at the end of the section here on line 49. I feel like it would be a bit redundant to also have similar phrasing here. That's just my opinion tho, let me know if you disagree.